dd-sdk-android/src/main/kotlin/com/datadog/android/rum/tracking/WhitelistPredicate.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.tracking
+
+/**
+ * Used to whitelist an Activity or Fragment in order to be considered as a RUM View Event in our
+ * Activity or Fragment view tracking strategies.
+ * @see FragmentViewTrackingStrategy
+ * @see ActivityViewTrackingStrategy
+ */
+interface WhitelistPredicate<T> {
```

Comment:
WhitelistPredicate is a weird name here. 
Whitelist and Blacklist are both different Predicate strategy, but Whitelist gives the false impression that you can only whitelist items.
Predicate means "Decide who you accept or deny"
WhiteList means "Deny all except those on the list"
BlackList means "Accept all except those on the list".

You can rename it `ViewPredicate` maybe instead.